pkg/asset/installconfig/gcp/dns.go

```diff
@@ -13,6 +13,12 @@ import (
 	survey "gopkg.in/AlecAivazis/survey.v1"
 )
 
+// Zone represents a GCP cloud DNS zone
+type Zone struct {
+	ID   string
+	Name string
```

Comment:
`ID` seems okay, but the name should be DNSName.

Reply:
Updated.